### PR TITLE
KLS-1673 dea cta snippet

### PR DESCRIPTION
**Pull request by timothyPatterson**
This PR introduces a 'Related Content Call-to-Action' snippet with associated CMS panel and template view which utilises an existing card component. 

### Workflow

- [x] Ticket exists in Jira https://uktrade.atlassian.net/jira/software/projects/KLS/boards/359?selectedIssue=KLS-1673
- [x] Jira ticket has the correct status.
- [x] A clear/description pull request messaged added.

### Reviewing help

- [x] Explains how to test locally, including how to set up appropriate data
1. Create a 'Related Content CTA' snippet
Screenshot 2023-12-21 at 10 42 01

2. In a 'DetailPage' add a 'Related Content' block
Screenshot 2023-12-21 at 10 43 53

3. Up to 3 CTAs can be added in each block
Screenshot 2023-12-21 at 10 43 07

4. Cards are displayed at relevant position in page
Screenshot 2023-12-21 at 10 50 48

4a. Cards are horizontally scrollable at mobile breakpoints
Screenshot 2023-12-21 at 10 52 34

### Housekeeping

- [x] Frontend assets have been re-compiled.

### Merging

- [x] This PR can be merged by reviewers. (If unticked, please leave for the author to merge)

### CODE REVIEW

File: styles/pages/learn-to-export.scss [hunk 1]
```diff
@@ -336,3 +336,68 @@ li .lte-suptopic {
...
   @extend .govuk-body
 }
+
+.related-links-cta-container {
+  display: flex;
+  flex-direction: row;
+  width: 100%;
+
+  // horizontal scrolling of child elements
+  @include govuk-media-query($until: tablet) {
```
Comment (jrbarnes9) [inline styles/pages/learn-to-export.scss:346]: I think this bp needs to be `desktop` to match the bp where we lose the side bar.
```suggestion
  @include govuk-media-query($until: desktop) {
```

File: styles/pages/learn-to-export.scss [hunk 2]
```diff
@@ -336,3 +336,68 @@ li .lte-suptopic {
...
+    overflow-x: scroll;
+    -webkit-overflow-scrolling: touch;
+    overflow-y: hidden;
+    margin: 0;
+    width: 95vw;
+  }
+
+  @include govuk-media-query($from: 641px, $until: 1170px) {
```
Comment (jrbarnes9) [inline styles/pages/learn-to-export.scss:354]:
```suggestion
  @include govuk-media-query($from: desktop, $until: 1170px) {
```

File: styles/pages/learn-to-export.scss [hunk 3]
```diff
@@ -336,3 +336,68 @@ li .lte-suptopic {
...
+      }
+    }
+  }
+
+  a {
+    display: flex;
+    margin-right: 20px !important;
+    width: 230px;
+
+    @include govuk-media-query($from: 641px, $until: 1170px) {
```
Comment (jrbarnes9) [inline styles/pages/learn-to-export.scss:367]:
```suggestion
    @include govuk-media-query($from: desktop, $until: 1170px) {
```

File: learn/templates/learn/related_pages_cta.html [hunk 4]
```diff
@@ -0,0 +1,7 @@
+{% load get_cta_attributes from helpers %}
+<div class="related-links-cta-container">
+    {% for cta in block.value %}
+        {% get_cta_attributes cta as cta_attrs %}
+        {% include 'components/great/card.html' with tag_container_inner_class="related-links-cta-container__tag-inner-container" tag_container_outer_class="related-links-cta-container__tag-outer-container" container_class="related-links-cta-container__title-container" title=cta.link_text url=cta_attrs.link show_title_link=True heading_class=cta_attrs.heading_class heading_level="h2" classes="great-card--modifier" tag=cta_attrs.tag_description tag_icon=cta_attrs.tag_icon %}
```
Comment (lewis-coulson-dit) [inline learn/templates/learn/related_pages_cta.html:5]: `great-card--modifier` swap this with a more meaningful name

File: styles/great/_components.scss [hunk 5]
```diff
@@ -420,9 +421,10 @@ figure.great-blockquote {
 }
 
 .great-card--modifier {
```
Comment (lewis-coulson-dit) [inline styles/great/_components.scss:423]: change this here as well

File: styles/pages/learn-to-export.scss [hunk 6]
```diff
@@ -336,3 +336,81 @@ li .lte-suptopic {
...
+    overflow-x: scroll;
+    -webkit-overflow-scrolling: touch;
+    overflow-y: hidden;
+    margin: 0;
+    width: 100vw;
+  }
+
+  @include govuk-media-query($from: desktop, $until: 1170px) {
+    & {
+      .great-card--modifier{
```
Comment (lewis-coulson-dit) [inline styles/pages/learn-to-export.scss:357]: and here

File: styles/pages/learn-to-export.scss [hunk 7]
```diff
@@ -336,3 +336,81 @@ li .lte-suptopic {
...
+    width: 230px;
+
+    @include govuk-media-query($from: desktop, $until: 1170px) {
+      width: unset;
+    }
+  }
+
+  & {
+    .great-position-relative{
+      position: relative;
```
Comment (lewis-coulson-dit) [inline styles/pages/learn-to-export.scss:374]: you shouldn't need this line

**Review by lewis-coulson-dit — APPROVED** [review]

**Pull request merged**